app/src/main/java/com/example/notes2text/usecases/FolderFactory.java

```diff
@@ -0,0 +1,15 @@
+package com.example.notes2text.usecases;
+
+import java.io.File;
+
+public class FolderFactory {
```

Comment:
Maybe put docstring on public classes as well.

Reply:
Since there are no changes to the state of the class and no parameter for the class, docstring is omitted for simplicity.